lib/country_state_select/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module CountryStateSelect
-  VERSION = '3.1.5'
+  VERSION = '3.2.0'
```

Comment:
Thoughts on `3.2.0` vs `3.1.6`?

Reply:
https://semver.org/ says that for `MAJOR.MINOR.PATCH`, increase the `MINOR` version when you add functionality in a backwards compatible manner, which you've done. `PATCH` is more for bug fixes. I think `3.2.0` makes sense.